content/the-ember-times-issue-192.md

```diff
@@ -0,0 +1,157 @@
+---
+title: The Ember Times - Issue No. 192
+authors:
+  - chris-ng
+  - bmish
```

Comment:
@jaredgalanis Before we merge the pull request, let's make sure to create a Markdown file for Bryan in the `author` folder.